guacamole-common/src/main/java/org/apache/guacamole/websocket/GuacamoleWebSocketTunnelEndpoint.java

```diff
@@ -109,15 +113,15 @@ public void onOpen(final Session session, EndpointConfig config) {
             // Get tunnel
             tunnel = createTunnel(session, config);
             if (tunnel == null) {
-                closeConnection(session, GuacamoleStatus.RESOURCE_NOT_FOUND);
+                closeConnection(session, GuacamoleStatus.RESOURCE_NOT_FOUND, null);
```

Comment:
Assuming that we do go the status code / WebSocket code route (ie: same as the HTTP case), the DRY approach here would be to overload `closeConnection()` with a variant which accepts only `GuacamoleStatus`, pulling the codes it needs from there.

Reply:
Sounds good.

Reply:
I re-implemented this with just `int` and no overloading, since the only reason to pass the `GuacamoleStatus` parameter is to access the `getGuacamoleStatusCode()` method.  Let me know how this looks - it doesn't feel very elegant, but maybe better than `Integer` and `null` route.

Reply:
Okay, dense moment is passed - I know what to do, now...hang on, another change coming.